### PR TITLE
RSDK-3603-consolidate-lua-files

### PR DESCRIPTION
**Pull request by nicksanford**
[Ticket](https://viam.atlassian.net/browse/RSDK-3603)
This is work required to unblock: https://github.com/viamrobotics/viam-cartographer/pull/127/files

Manually tested on both a PI appimage & aarm64 osx

### CODE REVIEW

**Review by kim-mishra — APPROVED** [review]
Please verify that the diff between the new files and the files in `iam-cartographer/cartographer/configuration_files/` are the same, otherwise LGTM!

**Comment by nicksanford** [comment]
> Please verify that the diff between the new files and the files in `iam-cartographer/cartographer/configuration_files/` are the same, otherwise LGTM!

```
pwd
/Users/nicksanford/viam-cartographer/viam-cartographer/lua_files

shasum map_builder.lua
d586804ef43dd6331e4dcc1c675d32c16457b386  map_builder.lua

shasum map_builder_server.lua
846162abddf0ad1bc0a22bf6e300f0487263c429  map_builder_server.lua

shasum pose_graph.lua
2bad71da5061aa1cef8dad915247c0c6631d98c6  pose_graph.lua

shasum trajectory_builder.lua
fcf8e6ac8e307a8da8c357a7caba1040a4fe8143  trajectory_builder.lua

shasum trajectory_builder_2d.lua
a43778d2c4de5cdb7ecc0a973baa46e9827ce6d7  trajectory_builder_2d.lua

shasum trajectory_builder_3d.lua
3a81ded4e5858c7ff6867947be938a9c4cdeab08  trajectory_builder_3d.lua
```

```
pwd
/Users/nicksanford/viam-cartographer/viam-cartographer/cartographer/configuration_files

shasum map_builder.lua
d586804ef43dd6331e4dcc1c675d32c16457b386  map_builder.lua

shasum map_builder_server.lua
846162abddf0ad1bc0a22bf6e300f0487263c429  map_builder_server.lua

shasum pose_graph.lua
2bad71da5061aa1cef8dad915247c0c6631d98c6  pose_graph.lua

shasum trajectory_builder.lua
fcf8e6ac8e307a8da8c357a7caba1040a4fe8143  trajectory_builder.lua

shasum trajectory_builder_2d.lua
a43778d2c4de5cdb7ecc0a973baa46e9827ce6d7  trajectory_builder_2d.lua

shasum trajectory_builder_3d.lua
3a81ded4e5858c7ff6867947be938a9c4cdeab08  trajectory_builder_3d.lua
```

File: viam-cartographer/lua_files/map_builder.lua [hunk 1]
```diff
@@ -0,0 +1,23 @@
+-- Copyright 2016 The Cartographer Authors
```
Comment (nicksanford) [inline viam-cartographer/lua_files/map_builder.lua:1]: Add comment that this file needs to be kept up with the cartographer project.

**Review by jeremyrhyde — APPROVED** [review]
LGTM. Thanks for adding the comment!

**Pull request merged**